templates/tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh.template

```diff
@@ -41,4 +41,5 @@
 
   # Cloning from a local path sets RepositoryUrl to a path and breaks Source Link.
   # Override RepositoryUrl to a URL to fix Source Link. The value doesn't matter.
-  dotnet build --configuration Debug Grpc.DotNet.sln -p:RepositoryUrl=https://github.com/grpc/grpc-dotnet.git
```

Comment:
ok, I see what might be the problem.
This file is a template - you might need to regenerate using tools/builgen/generate_projects.sh to actually update the tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh

Reply:
or just update that file manually (sanity tests will tell you if you did that correctly).